recharges/tests.py

```diff
@@ -376,3 +456,17 @@ def test_hotsocket_get_airtime_unrecoverable(self):
         # Check
         self.assertEqual(result.get(),
                          "airtime request for +277244555 is unrecoverable")
+
+    def test_Check_Hotsocket_Status(self):
+        recharge_id = self.make_recharge()
+        result = hotsocket_status(recharge_id)
+        self.assertEqual(result, "recharge is succesful")
+
+
+class TestRechrgeStatus(TaskTestCase):
+    def test_prep_hotsocket_status_dict(self):
```

Comment:
Move this test function into the previously used test class TestRechargeFunctions for consistency.